src/js/video.js

```diff
@@ -183,7 +185,7 @@ videojs.getComponent = Component.getComponent;
  */
 videojs.registerComponent = Component.registerComponent;
 
-/*
+/**
```

Comment:
The additional asterisk was left out intentionally by @mboles, in order to make the comment private. We should use the `@private` tag instead, at least until we figure out how our new jsdoc process will handle this type of object.